numba/tests/test_function_type.py

```diff
@@ -1072,3 +1072,28 @@ def bar_good(foos):
         self.assertRegex(
             str(cm.exception),
             r'.*first-class function call cannot use keyword arguments')
+
+    def test_issue_5615(self):
+        @njit
+        def foo1(x):
+            return x + 1
+
+        @njit
+        def foo2(x):
+            return x + 2
+
+        from numba import literal_unroll
```

Comment:
Think this needs to be a global (there's a bug, don't recall ticket, which makes `literal_unroll` as a freevar not work).

Reply:
Fixed the bug https://github.com/numba/numba/pull/5638

Reply:
Thanks @stuartarchibald, I will update this PR accordingly.

Reply:
Done.